examples/maven/build.gradle

```diff
@@ -8,7 +8,7 @@ task buildExamples(type: Exec, dependsOn: [':modules:testpackage:install', ':mod
 }
 
 task checkExamples(type: Exec, dependsOn: buildExamples) {
-    commandLine 'target/maven-example-1.0-SNAPSHOT.jar'
+    commandLine 'java', '-jar', 'target/maven-example-1.0-SNAPSHOT.jar'
```

Comment:
I'm not sure why this is now necessary - the test seems to rely on the JAR having the executable bit set, but the code does not do that and seems to have never done so. This can be changed in the code, but for now I've updated the test.